src/schnetpack/nn/electronic_embeeding.py

```diff
@@ -0,0 +1,123 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from typing import Optional
+from schnetpack.nn.residual_blocks import ResidualMLP
+
+class ElectronicEmbedding(nn.Module):
+    """
+    Single Head self attention block for updating atomic features through nonlocal interactions with the
+    electrons.
+
+    Arguments:
+        num_features (int):
+            Dimensions of feature space.
+        num_basis_functions (int):
+            Number of radial basis functions.
+        num_residual_pre_i (int):
+            Number of residual blocks applied to atomic features in i branch
+            (central atoms) before computing the interaction.
+        num_residual_pre_j (int):
+            Number of residual blocks applied to atomic features in j branch
+            (neighbouring atoms) before computing the interaction.
+        num_residual_post (int):
+            Number of residual blocks applied to interaction features.
+        activation (str):
+            Kind of activation function. Possible values:
+            'swish': Swish activation function.
+            'ssp': Shifted softplus activation function.
+    """
+
+    def __init__(
+        self,
+        num_features: int,
+        num_residual: int,
+        activation: str = "swish",
```

Comment:
change to `activation: Callable = ShiftedSoftplus(trainable=True)`

Reply:
done in c86b404 and f99a432